rubocop-patch.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
```

Comment:
recommended fix from here:
https://gist.github.com/lcd047/96138909015f2f8d2d36

It silences parses warnings due to ruby 2.1.x.  In theory, can just remove this patch when we upgrade ruby